recipes/pyproj/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Just did.  That is much cleaner!

(Hopefully we will re-invent the wheel completely with conda and have only one file like with RPM specfiles :wink: )